services/hanke-service/README.md

```diff
@@ -167,6 +167,11 @@ cd haitaton-backend
 .\scripts\HEL-GIS-data-import.cmd
 ```
 
+# Emails
+When running locally, when sending emails from the system the emails are sent to smtp4dev, which is
```

Comment:
When running locally, when sending emails from the system the emails are sent to smtp4dev, -> When running locally, the system the emails are sent to smtp4dev,